pywinauto/unittests/test_application_macos.py

```diff
@@ -31,18 +31,11 @@ def _test_app():
 sys.path.append(".")
 
 if sys.platform == 'darwin':
-    class ApplicationTestCases(unittest.TestCase):
-
-        """Unit tests for the application.Application class"""
-
-        def setUp(self):
-
-            pass
-
-        def tearDown(self):
-
-            pass
-
+    class ApplicationEspecialTestCases(unittest.TestCase):
+        """
+        Unit tests for the application.Application class
+        There are test cases which has unique start/end logic
+        """ 
```

Comment:
Even for these special cases it'd be better to have a proper cleanup. Remember that if test fails we don't want any leftovers. 
```python
def setUp(self):
    self.app = Application()

def tearDown(self):
    self.app.kill()
```